app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -1,89 +1,129 @@
 <template lang="pug">
-tm-page(:title='validatorTitle(this.validator)')
+tm-page
+  template(slot="menu-body", v-if="config.devMode"): tm-balance(:unstakedAtoms="user.atoms" :tabs="tabs")
   div(slot="menu"): tm-tool-bar
     router-link(to="/staking" exact): i.material-icons arrow_back
     anchor-copy(v-if="validator" :value="validator.owner" icon="content_copy")
 
-
   tm-data-error(v-if="!validator")
+
   template(v-else)
-    tm-part(title='!!! CRITICAL ALERT !!!' v-if="validator.revoked")
-      tm-list-item(title="This validator is revoked!" subtitle="Are you the owner? Go fix it!" type="anchor" href="https://cosmos.network/docs/validators/validator-setup.html#common-problems")
-
-    tm-part#validator-profile(title='Validator Profile')
-      img.avatar(v-if="validator.keybase" :src="validator.keybase.avatarUrl" width="192" height="192")
-      img.avatar(v-else src="~assets/images/validator-icon.svg" width="192" height="192")
-      .list-items
-        tm-list-item(dt="Moniker" :dd="validator.description.moniker")
-        tm-list-item(v-if="validator.keybase" dt="Keybase" :dd="validator.keybase.userName" :href="validator.keybase.profileUrl")
-        tm-list-item(v-else dt="Keybase" dd="add your avatar" href="https://cosmos.network/docs/validators/validator-setup.html#edit-validator-description" target="_blank")
-        tm-list-item(dt="Website" :dd="validator.description.website")
-        tm-list-item(dt="Details" :dd="validator.description.details")
-
-    tm-part(title='Validator Keys')
-      tm-list-item(dt="Owner" :dd="validator.owner")
-      tm-list-item(dt="Pub Key" :dd="validator.pub_key")
-
-    tm-part(title='Validator Stake' v-if="!validator.revoked")
-      tm-list-item(dt="Voting Power" :dd="pretty(validator.voting_power) + ' ' + this.config.bondingDenom")
-      tm-list-item(dt="Self Bonded" :dd="pretty(selfBond) + ' ' + this.config.bondingDenom")
-      tm-list-item(dt="Bond Height" :dd="`Block ${validator.bond_height}`")
-
-    tm-part(title='Commission')
-      tm-list-item(dt="Commission" :dd="pretty(validator.commission) + ' %'")
-      tm-list-item(dt="Commission Maximum" :dd="pretty(validator.commission_max) + ' %'")
-      tm-list-item(dt="Commission Change-Rate" :dd="pretty(validator.commission_change_rate) + ' %'")
-      tm-list-item(dt="Commission Change Today" :dd="pretty(validator.commission_change_today) + ' %'")
+    .validator-profile__header.validator-profile__section
+      .column
+        img.avatar(v-if="validator.keybase" :src="validator.keybase.avatarUrl")
+        img.avatar(v-else src="~assets/images/validator-icon.svg")
+      .column.validator-profile__header__info
+        .row.validator-profile__header__name
+          .column
+            div.validator-profile__status-and-title
+              span.validator-profile__status(v-bind:class="statusColor" v-tooltip.top="status")
+              .validator-profile__header__name__title {{ validatorTitle(validator) }}
```

Comment:
Can we get rid of the `... - (address)` please ? I don't see a point on keeping it since we're already showing the validator's owner address

Reply:
no i don't think that's a good idea. the validator profile is like a blown up version of the validator Li - i want to keep it for consistency and familiarity. the form below (validator details) should show the full address.

Reply:
Fede meant the suffix after the title. For consistency with the LiValidator I removed it.

Reply:
Well according to the [design](https://app.zeplin.io/project/59e66beff6721ea8bf562e5b/screen/5b8d49761076bc0ce6e551b1) we shouldn't be showing the `- (address)` ...